fair_mango/metrics/metrics.py

```diff
@@ -697,3 +700,280 @@ def is_biased(self, threshold: float = 0.1) -> dict[str, bool]:
                 bias[target] = False
 
         return bias
+
+
+def super_set_fairness_metrics(
```

Comment:
maybe create a separate file for superset metrics

Reply:
Good idea! I will transfer the two superset functions to a new file as suggested.